ui/imports/shared/popups/SendModal.qml

```diff
@@ -159,10 +166,14 @@ StatusDialog {
                 inverted: true
             }
         }
-        selectedAccount: popup.selectedAccount
-        changeSelectedAccount: function(newAccount) {
-            popup.selectedAccount = newAccount
-            assetSelector.selectedAsset = store.getAsset(selectedAccount.assets, assetSelector.selectedAsset.symbol)
+        selectedAccount: !!popup.selectedAccount ? popup.selectedAccount: {}
+        chainShortNames: store.getAllNetworksSupportedString()
+        onSelectedIndexChanged: d.setSelectedAccount()
+        Connections {
+            target: popup.store.accounts
+            function onModelReset() {
```

Comment:
Connecting to `modelReset` signal is something we should avoid (usually whenever you want to do this check if there is some better solution), but this most likely comes from unclear intention is selected account maintained by the backend or qml. 

Thinking now... do we need this connection at all? When is the case when this will be triggered? 
As I see it's called when accounts get any change, but I guess (correct me if I am wrong) that is completely irrelevant if you've already opened send popup? Cause at the moment when user opens send modal it should have in its offer accounts added till that moment, while the send modal is opened the only way a new account to be added to a wallet is if you have paired device and add new account there, but in that case we should not care about it.

Reply:
hey @saledjenic , so the use case where this impacts actually sending, 
1. is if the SendModal is open and
2.  we hit the 10 mins timer to refresh account & assets list and
3.  if the account selected in SendModal has different token amounts now, maybe not enough to send the tx or many so that tx is  not possible, all of this will not reflect unless we react to the modelReset


The modelReset Connection had to be added to listen and update the assets for the selected account in case there was an update from the backend.

Reply:
@Khushboo-dev-cpp yes, I understand the need now, thanks.

Few things:
- Even if the token amount is enough for sending a transaction while user sets all parameters he wants the state on a blockchain may be changed but we won't get it till another "account refresh" and even the app says that user has enough assets to perform an action executing a transaction won't be possible. So that's not that reliable. 
- Now from the code/architectural point connecting to the `modelReset` signal in order to perform some other action is not something common, it does make me think that something is wrong in a logical order, looks like we're mixing what's backend what's qml responsibility. Imagine that while you're here a new sync signal arrives which removes or adds an account or when we introduce D&D feature changes the order of current accounts list, that will "refresh accounts" and currently stored index on the qml side will point either to a wrong account or maybe to the index which is out of the list size. That means in the first case after account refresh an account user previously selected will change (all of a sudden by user's point of view) or we will get an index out of range error in second case.
- If you ask me I would just remove this connection and leave send popup to reflects the current state of the app at the moment that send popup is opened.

To be clear, I know that many things I am speaking of here were already here and made you make these changes but want to discuss them.

Reply:
How about calling `setSelectedAccount` on count changed or currentIndex instead of model reset? Usually, these two properties are invalidated in an account reset.

Reply:
> p is opened.

would still be a work around

Reply:
As discussed with Anthony, we'll tackle this separately. This PR is ready for review now :)